docs/04. Seguimiento/4.8. Registro de cambios.mdx

```diff
@@ -24,29 +25,40 @@ planificación inicial.
+| 3  | Poder importar/exportar familias desde excel| <span style={{backgroundColor: 'red', display: 'inline-block', width: '100%', height: '100%', textAlign: 'center'}}>Alta</span>     | 11 de abril de 2024         | Incluido en la planificacion                                                           | Sí                           | Sí                             |<span style={{color: 'orange'}}>En progreso</span>|
+| 4  | Calcular la edad de los miembros a partir de su fecha de nacimiento y mostrarla | <span style={{backgroundColor: 'orange', display: 'inline-block', width: '100%', height: '100%', textAlign: 'center'}}>Media</span>     | 11 de abril de 2024         | Incluido en la planificacion                                                           | Sí                           | Sí                             |<span style={{color: 'orange'}}>En progreso</span>|
+| 5  | Actualizar correo electrónico de contacto a "tertuliacirioycostal@gmail.com" | <span style={{backgroundColor: 'green', display: 'inline-block', width: '100%', height: '100%', textAlign: 'center'}}>Baja</span>     | 11 de abril de 2024         | Incluido en la planificacion                                                           | Sí                           | Sí                             |<span style={{color: 'orange'}}>En progreso</span>|
+| 6  | Mostrar estadísticas requeridas por el banco de alimentos | <span style={{backgroundColor: 'red', display: 'inline-block', width: '100%', height: '100%', textAlign: 'center'}}>Alta</span>     | 11 de abril de 2024         | Incluido en la planificacion                                                           | Sí                           | Sí                             |<span style={{color: 'orange'}}>En progreso</span>|
+| 7  | Mostrar mensajes de error en el formulario de login | <span style={{backgroundColor: 'orange', display: 'inline-block', width: '100%', height: '100%', textAlign: 'center'}}>Media</span>     | 11 de abril de 2024         | Incluido en la planificacion                                                           | Sí                           | Sí                             |<span style={{color: 'orange'}}>En progreso</span>|
+| 8  | Incluir los campos número de citas totales, edad y si el servicio que tenían que proporcionarles está finalizado en el listado de beneficiarios | <span style={{backgroundColor: 'orange', display: 'inline-block', width: '100%', height: '100%', textAlign: 'center'}}>Media</span>    | 11 de abril de 2024         | Incluido en la planificacion                                                           | Sí                           | Sí                             |<span style={{color: 'orange'}}>En progreso</span>|
```

Comment:
This is already covered by the list of interventions on the right panel of a beneficiary detail page and the option to "Finalize" someone

Reply:
I deleted it